pkg/net/net.go

```diff
@@ -2,11 +2,23 @@ package net
 
 import (
 	"context"
-
 	"github.com/keep-network/keep-core/pkg/internal/pb"
 	"github.com/keep-network/keep-core/pkg/operator"
 )
 
+// RetransmissionStrategy represents a specific retransmission strategy.
+type RetransmissionStrategy int
+
+const (
+	// StandardRetransmissionStrategy is the default retransmission strategy
+	// that retransmit the message with a constant frequency.
+	StandardRetransmissionStrategy RetransmissionStrategy = iota
+	// BackoffRetransmissionStrategy is a retransmission strategy that
+	// retransmit the message with an exponentially increasing delay between
```

Comment:
```suggestion
	// retransmits the message with an exponentially increasing delay between
```

Reply:
Addressed in https://github.com/keep-network/keep-core/pull/3392/commits/a1a9ec4f552bc11bb7ee10c1436d90076b42fa05